cli/src/main/java/com/box/l10n/mojito/cli/command/L10nJCommander.java

```diff
@@ -92,6 +97,32 @@ public void init() {
     createJCommanderForRun();
   }
 
+  private void notifyFailure(Command command, String[] args, String errorMessage) {
+    if (!Strings.isNullOrEmpty(command.getFailureSlackNotificationChannel())) {
+      if (this.slackClient != null) {
```

Comment:
nit: You can include this in the previous if statement to avoid the double if

Reply:
Done